benchflow-test-manager/application/src/test/java/cloud/benchflow/testmanager/helpers/TestBundle.java

```diff
@@ -88,17 +85,19 @@ public static ZipInputStream getValidTestBundleZip(TemporaryFolder temporaryFold
   public static File getMissingTestDefinitionTestBundleFile(TemporaryFolder temporaryFolder)
       throws IOException {
 
-    File tempDir = temporaryFolder.newFolder();
+    // setup the bundle contents without test definition
+    ZipEntrySource[] addedEntries = new ZipEntrySource[] {
+        new FileSource(TestFiles.getTestDeploymentDescriptorFile().getName(),
+            TestFiles.getTestDeploymentDescriptorFile()),
+        new FileSource("models/" + TestFiles.getModelsFolderFile().listFiles()[0].getName(),
```

Comment:
@jeppe-style you might want to update this so that accessing to the single model file is not hardcoded.